app/controllers/needs_controller.rb

```diff
@@ -8,14 +8,20 @@ class NeedsController < ApplicationController
   helper_method :get_param
 
   def index
-    @params = params.permit(:assigned_to, :status, :category, :page, :order_dir, :order, :commit, :is_urgent)
+    @params = params.permit(:assigned_to, :status, :category, :page, :order_dir, :order, :commit, :is_urgent, :created_by_me)
     @users = User.all
```

Comment:
I am not sure if we still need there @users and @roles variables here

Reply:
Good spot - these have been superceded by the 'construct_assigned_to_options' method, will remove them